providers/ms365/resources/builtin_applications.go

```diff
@@ -0,0 +1,1868 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package resources
+
+const (
+	MicrosoftEntraTenantID = "f8cdef31-a31e-4b4a-93e4-5f571e91255a"
+	MicrosoftTenantID      = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+)
+
+type KnownApplicationId struct {
+	Name  string
+	AppId string
+}
+
+// Apps owned by "f8cdef31-a31e-4b4a-93e4-5f571e91255a"
+// https://learn.microsoft.com/en-us/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in#application-ids-of-commonly-used-microsoft-applications
```

Comment:
Is there any plan for how we'll keep this updated over time?

Reply:
Replaced with new implementation. Found another way to implement this more reliably.